Nightly dev builds failing

From 04/07/23, the nightly dev builds in github actions began failing in the "Decrpyt a secret" step with the following error:
```bash
gpg: gcry_kdf_derive failed: Invalid data
gpg: decryption failed: No secret key
Error: Process completed with exit code 2.
```

For example: https://github.com/DiamondLightSource/httomo/actions/runs/5686700912/job/15413946524

--- a/.scripts/conda_upload.sh
+++ b/.scripts/conda_upload.sh
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-PKG_NAME=httomo
-USER=httomo-team
+PKG_NAME=httomolib
+USER=DLS
 OS=linux-64
 CONDA_TOKEN=$(cat $HOME/.secrets/my_secret.json)
 
@@ -9,7 +9,7 @@ mkdir ~/conda-bld
 conda config --set anaconda_upload no
 export CONDA_BLD_PATH=~/conda-bld
 
-#for python_ver in 3.9 3.10; do
+#for python_ver in 3.9 3.10; do   
 #    for numpy_ver in 1.21 1.22 1.23 1.24; do
 #        export VERSION=`date +%Y.%m`"_py"$python_ver"_np"$numpy_ver
 #        conda build . --numpy $numpy_ver --python $python_ver
@@ -19,7 +19,7 @@ export CONDA_BLD_PATH=~/conda-bld
 $CONDA/bin/conda install conda-build
 $CONDA/bin/conda install -c anaconda anaconda-client
 
-$CONDA/bin/conda build . -c conda-forge -c https://conda.anaconda.org/httomo/
+$CONDA/bin/conda build . -c conda-forge -c httomo
 
 # upload packages to conda
 find $CONDA_BLD_PATH/$OS -name *.tar.bz2 | while read file

